demo/src/plot_demo.rs

```diff
@@ -304,6 +334,14 @@ impl LineDemo {
     }
 }
 
+fn interpolate(start: Color32, end: Color32, y: f64) -> Result<Color32, TryFromIntError> {
+    Ok(Color32::from_rgb(
+        u8::try_from((start.r() as f64 + y * (end.r() as f64 - start.r() as f64)) as u64)?,
+        u8::try_from((start.g() as f64 + y * (end.g() as f64 - start.g() as f64)) as u64)?,
+        u8::try_from((start.b() as f64 + y * (end.b() as f64 - start.b() as f64)) as u64)?,
+    ))
+}
+
```

Comment:
This could all be replaced with https://docs.rs/epaint/latest/epaint/struct.Color32.html#method.lerp_to_gamma

Reply:
Nice. Fixed.